services/common/src/components/forms/RenderMultiSelect.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { FC } from "react";
+import { WrappedFieldProps } from "redux-form";
+import { Form } from "@ant-design/compatible";
+import "@ant-design/compatible/assets/index.css";
+import { Select } from "antd";
+import { caseInsensitiveLabelFilter } from "@mds/common/redux/utils/helpers";
+
+/**
+ * @constant RenderSelect - Ant Design `Select` component for redux-form - used for small data sets that (< 100);
+ */
+
+interface MultiSelectProps extends WrappedFieldProps {
```

Comment:
this one had a couple of changes to satisfy TS/redux-form. Making meta and I think data mandatory, and that WrappedFieldProps.